src/NuGet.Clients/NuGet.VisualStudio.Implementation/Utility/FrameworkNameUtility.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Runtime.Versioning;
+using NuGet.Frameworks;
+
+namespace NuGet.VisualStudio.Implementation.Utility
+{
+    public static class FrameworkNameUtility
```

Comment:
Alternatives? This is a sort of mapping that is now needed on two different of our IVS interfaces. Best to have the mapping in one place, right?